engine/src/object.h

```diff
@@ -374,7 +380,8 @@ class MCObject : public MCDLlist
 	
 	virtual void timer(MCNameRef mptr, MCParameter *params);
 	virtual uint2 gettransient() const;
-	virtual void setrect(const MCRectangle &nrect);
+	virtual void doSetRect(const MCRectangle &nrect);
+	virtual void setrect(const MCRectangle &p_rect);
```

Comment:
Does this still have to be virtual if it is now a wrapper which prompts a geomchanged?